benchmarks/distributed/rpc/rl/coordinator.py

```diff
@@ -0,0 +1,79 @@
+import torch
+import torch.distributed.rpc as rpc
+from torch.distributed.rpc import rpc_async, rpc_sync, remote
+
+from Agent import AgentBase
+from Observer import ObserverBase
+
+import copy
+import time
+
+COORDINATOR_NAME = "coordinator"
+AGENT_NAME = "agent"
+OBSERVER_NAME = "observer{}"
+
+
+def call_method(method, rref, *args, **kwargs):
+    # a helper function to call a method on the given RRef
+    return method(rref.local_value(), *args, **kwargs)
+
+
+def remote_method(method, rref, *args, **kwargs):
+    # a helper function to run method on the owner of rref and fetch back the result using RPC
+    args = [method, rref] + list(args)
+    return rpc_sync(rref.owner(), call_method, args=args, kwargs=kwargs)
```

Comment:
You should be able to get rid of these by using RRef helpers: 

https://pytorch.org/docs/master/rpc.html#torch.distributed.rpc.RRef.rpc_sync